### DISCUSSION TITLE
How do I check on which worker node particular entry is stored? How do I check how much memory each worker node is using?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#22931] [Q&A and General discussion]
How do I check on which worker node particular entry is stored? How do I check how much memory each worker node is using?

**Comment by yhmo** [comment]
If you deploy the Milvus by k8s, you can deploy a monitoring service to monitor Milvus: https://milvus.io/docs/monitor.md

**Comment by yhmo** [comment]
The monitoring service can show you how much memory usage for each node. But it can't tell you which node a particular entry is stored. In fact, there is no API to tell where a particular entry is stored.

**Reply by martinenkoEduard** [reply]
because it is stored in segments?

**Reply by yhmo** [reply]
> because it is stored in segments?

Yes, segments are invisible to users.
